ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge.cfg

```diff
@@ -132,6 +132,7 @@ router path-selection
          stun server-profile MPLS-cv-pathfinder-pathfinder-Ethernet2
       !
       peer dynamic
+         ipsec disabled
```

Comment:
Why is this only getting added on some devices and not others? (like autovpn devices are not getting this)

Reply:
I have rendered this only for devices which has the local path group. so if a device doesn't have MPLS path group, it won't be rendered for it.